assets/app/views/projects.html

```diff
@@ -1,7 +1,13 @@
 <div class="container">
   <h1>Projects</h1>
   <div class="tile tile-project tile-click" style="margin-bottom: 10px;" ng-repeat="project in projects">
-    <h2 class="project"><a class="tile-target" href="project/{{project.metadata.name}}">{{project.displayName || project.metadata.name}}</a></h2>
+    <h2 class="project">
+      <a class="tile-target" href="project/{{project.metadata.name}}">{{project.displayName || project.metadata.name}}</a>
+      <span class="pficon-layered" data-toggle="tooltip" data-placement="right" title="This project has been marked for deletion." style="cursor: help; vertical-align: top; margin-left: 5px;" ng-if="project.status.phase != 'Active'">
```

Comment:
put the `ng-if="project.status.phase != 'Active'"` as the first attribute? otherwise it's easy to miss that the block is conditional